dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/channel/internal/HttpServiceContextImpl.java

```diff
@@ -2853,7 +2854,19 @@ final protected void sendFullOutgoing(WsByteBuffer[] wsbb, HttpBaseMessageImpl m
                 }
 
             } else if (msg.isChunkedEncodingSet()) {
-                createEndOfBodyChunk();
+                HttpInboundServiceContextImpl localHisc = null;
+                if (this instanceof HttpInboundServiceContextImpl) {
+                    localHisc = (HttpInboundServiceContextImpl) this;
+                }
+                if (localHisc != null && localHisc instanceof HttpInboundServiceContextImpl
```

Comment:
Is the  localHisc instanceof HttpInboundServiceContextImpl   check needed? It was checked above before setting localHisc